opus/application/static_media/js/browse.js

```diff
@@ -1196,6 +1200,53 @@ var o_browse = {
         return url;
     },
 
+    // check the cache of both rendered elements and variable o_browse.galleryData; remove 'far away' observations
+    // from cache to avoid buildup of too much data in the browser which slows things down
+    // Two functions; one to delete single elements, just a tool for the main one, manageObservationCache, to loop.
+    manageObservationCache: function(count, append) {
+        let tab = `#${opus.prefs.view}`;
+        var galleryObsElem =  $(`${tab} .gallery [data-obs]`);
+        var tableObsElem = $(`${tab} .op-dataTable-view [data-obs]`);
+
+        function deleteCachedObservation(index) {
```

Comment:
So how does this work for obs that are now in the cart but in the future we remove them from the cart? Do they end up sitting around forever because there's no reference to them in the gallery and thus we will never try to remove them?

Reply:
good point.  I suppose we have several choices:
-leave them hanging around <seems not fabulous but truthfully may not matter in the bigger scheme of things>
-when we remove things from the cart, check to see if these are orphaned observations and remove if so <is it really worth the trouble....?  ..premature optimization comes to mind>
-create a separate global var for the cart, so the info is managed independently.  If we did this, then we might as well allow different sorting for browse vs. cart.